micmac_li3ds/api.py

```diff
@@ -402,15 +402,15 @@ def update_obj(args, metadata, obj, type_):
         obj.setdefault('description', 'Imported from "{basename}"')
     if type_ in args:
         obj.update({k: v for k, v in args[type_].items() if v})
-    for key in obj:
-        if obj[key]:
+    for key in list(obj.keys()):
```

Comment:
I just want to be able to delete keys as I iterate over the `obj` dict. `list(obj.keys())` gives me a new list with the keys.